Core/GDCore/Project/EventsFunctionsExtension.h

```diff
@@ -20,7 +20,9 @@ class Project;
 }  // namespace gd
 
 namespace gd {
-
+// TODO Remove the EventsFunctionsContainer extension and make it an attribute.
```

Comment:
I think you mean the inheritance?
```suggestion
// TODO Remove the EventsFunctionsContainer inheritance and make it an attribute.
```